### PR TITLE
Replaces in game references of rig with hardsuit

### PR DESCRIPTION
**Pull request by Alberyk**
What it says in the title.

### CODE REVIEW

File: code/datums/uplink/hardsuit_modules.dm [hunk 1]
```diff
@@ -38,3 +38,9 @@
 	name = "Mounted Laser Cannon"
 	item_cost = 16
 	path = /obj/item/rig_module/mounted
+
+/datum/uplink_item/item/tools/rig_cooling_unit
```
Comment (NonQueueingMatt) [inline code/datums/uplink/hardsuit_modules.dm]: the path name could use some changes too

File: code/datums/uplink/hardsuit_modules.dm [hunk 2]
```diff
@@ -38,3 +38,9 @@
...
+	name = "mounted suit cooling unit"
+	item_cost = 8
+	path = /obj/item/rig_module/cooling_unit
```
Comment (NonQueueingMatt) [inline code/datums/uplink/hardsuit_modules.dm]: ditto

File: code/datums/uplink/hardsuit_modules.dm (hunk 1 above)
Comment (Alberyk) [reply code/datums/uplink/hardsuit_modules.dm]: This is not inside the scope of the pr, rig is still used in many places for the code. Hardsuit should be the in game name.

Comment (NonQueueingMatt) [reply code/datums/uplink/hardsuit_modules.dm]: fair

**Comment by alsoandanswer** [comment]
should probably add a TODO on renaming rig paths to hardsuit for codesistency

**Comment by NonQueueingMatt** [comment]
nah, todos are always forgotten

**Pull request merged**